src/web/app/login-page.component.html

```diff
@@ -0,0 +1,88 @@
+        </div>
+      </div>
+    </div>
+    <div *ngIf="isLogInWithEmail">
+      <div class="card-header">
+        <strong>Log In with Email</strong>
+      </div>
+      <div class="card-body">
+        <form [formGroup]="formLogin" (ngSubmit)="logInWithEmail(formLogin)">
+          <div class="form-group row">
+            <label class="col-sm-2 col-form-label">
+              <strong>Email:</strong>
+            </label>
+            <div class="col-sm-10">
+              <input class="form-control" type="email" formControlName="email" maxlength="254" email="true" spellcheck="false">
+            </div>
+          </div>
+          <div *ngIf="captchaSiteKey !== ''" class="form-group row">
+            <div class="col-sm-2"></div>
+            <div class="col-sm-10">
+              <ngx-recaptcha2 #captchaElem
+                              [siteKey]="captchaSiteKey"
+                              (success)="handleCaptchaSuccess($event)"
+                              [useGlobalDomain]="false"
+                              [size]="size"
+                              [hl]="lang"
+                              formControlName="recaptcha">
+              </ngx-recaptcha2>
+            </div>
+          </div>
+          <div class="form-group row">
+            <div class="col-sm-2"></div>
+            <div class="col-sm-10">
+              <button type="submit" class="btn btn-primary" [disabled]="isLoggingInWithEmail">
+                <tm-ajax-loading *ngIf="isLoggingInWithEmail"></tm-ajax-loading>Login
+              </button>
+              <button class="btn btn-light ms-2" (click)="isLogInWithEmail=false; isLoginPage=true">Cancel</button>
+            </div>
+          </div>
+        </form>
+      </div>
+    </div>
+    <div *ngIf="isLoginEmailSent">
+      <div class="card-header">
+        <strong>Login Email Sent</strong>
+      </div>
+      <div class="card-body">
+        <p>A login email with additional instructions was sent to {{formLogin.controls['email'].value}}.<br>Check your email to complete login.</p>
+        <a href="#" style="text-decoration: underline;" (click)="isTroubleGettingEmail=true; isLoginEmailSent=false;">
+          Trouble getting email?
+        </a>
+      </div>
+    </div>
+    <div *ngIf="isTroubleGettingEmail">
+      <div class="card-header">
+        <strong>Trouble Getting Email</strong>
+      </div>
+      <div class="card-body">
+        <p>Try these common fixes:</p>
+        <ul>
+          <li>Check if the email was marked as spam or filtered.</li>
+          <li>Check your internet connection.</li>
+          <li>Check that you did not misspell your email.</li>
+          <li>Check that your inbox space is not running out or other inbox settings related issues.</li>
+        </ul>
+        <p class="mt-3">If the steps above didn't work, you can resend the email. Note that this will deactivate the link in the older email.</p>
+        <div>
```

Comment:
A big issue here is the tone and content of the message. This message suggests to the user "this is your fault, here are a list of things you can do to fix it", which is not the message we want to send.

"Check your internet connection." and "Check that your inbox space is not running out or other inbox settings related issues." is additional noise at best and condescending at worst. These are not helpful to the user and should be removed. 

"Check that you did not misspell your email" also is not great, because the user has no way to of checking their email from this panel. Also this emphasises that the user is the one that made the mistake, basically saying please go back and check that you didn't mess up.

**Suggestion:**

We have sent an email to **email[at]email.com**.

If you haven't received this within a few minutes, it may have landed in your spam folder. Still having trouble? Try resending this email by clicking the resend button below.

Resend Cancel